### PR TITLE
Add bio fields

### PR DESCRIPTION
**Pull request by imncls**
I don't know this matches Glitch-soc's code style...
And I haven't deleted the current Glitch-soc profile metadata.
Because I couldn't find those codes.
I'm sorry.

| Before | After |
| :-----: | :----: |
|  |  |

| Before | After |
| :-----: | :----: |
|  |  |

| Before | After |
| :-----: | :----: |
|  |  |

### CODE REVIEW

File: app/javascript/flavours/glitch/styles/accounts.scss [hunk 1]
```diff
@@ -573,4 +573,58 @@
   }
 }
 
+.account__header__fields {
```
Comment (ThibG) [inline app/javascript/flavours/glitch/styles/accounts.scss]: You could reuse the style for `.metadata` instead.

File: app/javascript/flavours/glitch/styles/components/accounts.scss [hunk 2]
```diff
@@ -509,3 +509,40 @@
     margin-bottom: 0;
   }
 }
+
+.account__header .account__header__fields {
```
Comment (ThibG) [inline app/javascript/flavours/glitch/styles/components/accounts.scss]: You could reuse the style for `.account_metadata` instead.

File: app/javascript/flavours/glitch/styles/accounts.scss (hunk 1 above)
Comment (ThibG) [reply app/javascript/flavours/glitch/styles/accounts.scss]: Oh, and by reuse, I mean, maybe we could change the name of the existing CSS class to `account__header__fields` and change the glitch-soc-specific code to use that new name.

**Comment by imncls** [comment]
glitch-soc-style metadata and tootsuite-style metadata are almost the same now, and tootsuite-style is is better.
So, I think that it is better to delete glitch-soc-style metadata to make a not complicated.
Can I get your opinion?

**Comment by yipdw** [comment]
If we can keep glitch-soc rendering around without a large complexity cost, I think it would be a good idea to do that.  Keeping it around means that existing profiles continue to render the way they've rendered.  If people want to adjust their profiles to use tootsuite's implementation, that's fine, but I think that should be their choice.

If there is a very large cost to keeping both systems in place (i.e. the renderer can't actually handle both without a very complicated decision process) then I think it is acceptable to drop glitch-soc's rendering in favor of tootsuite's.

**Comment by ThibG** [comment]
The only issue I have with tootsuite-style bio feeds is that you're limited to 4 fields, which is kinda arbitrary.
I'm not too sure about removing glitch-style metadata handling just yet, but I'd be in favor of skipping them whenever tootsuite-style bio fields are used.

**Comment by yipdw** [comment]
Oh, this is about rendering both glitch-soc and tootsuite metadata; I missed that.  Yes, I am also in favor of prioritizing tootsuite-style bio fields over glitch-soc-style bio fields.

**Pull request closed without merging**